client/submitAssignment.js

```diff
@@ -173,7 +211,10 @@ function validateRunCmd() {
 
 let argCount = 1;
 
-// Adds key and argument text boxes on click of "Add" button
+/**
+ * Appends a new div element with two text boxes to the 'cmdArgs' HTML element
```

Comment:
Again the comment is too implementation specific.

Reply:
Okay.